barretenberg/cpp/src/barretenberg/stdlib/transcript/transcript.test.cpp

```diff
@@ -120,6 +123,108 @@ TEST(RecursiveHonkTranscript, InterfacesMatch)
     EXPECT_TRUE(CircuitChecker::check(builder));
 }
 
+TEST(RecursiveHonkTranscript, Infinity)
+{
+    using NativeGroup = typename grumpkin::g1;
+    using NativeCommitment = typename NativeGroup::affine_element;
+
+    // using Group = cycle_group<Builder>;
```

Comment:
remove

Reply:
oh i forgot about these changes and missed them when doing a round of cleanup, mb, thanks for pointing out